src/_data/sidenav.yml

```diff
@@ -60,6 +60,16 @@
 
 - title: App solutions
   children:
+    - title: AI
+      children:
+      - title: Overview
+        permalink: /resources/ai-overview
+      - title: generative_ai package
+        permalink: https://pub.dev/packages/google_generative_ai
+      - title: Build with Google AI Dart SDK (video)
+        permalink: https://www.youtube.com/watch?v=1AuzJEiHjO4
+      - title: Google AI Studio
```

Comment:
Does it make sense to have Google AI Studio here? Not sure if it's a direct resource for building with Flutter. I can be swayed either way, but if we keep - can we make the title a bit more descriptive? "Google AI Studio: for prototyping AI prompts"